### ISSUE TITLE
Symlinks in static-directory are not copied to output directory

### ISSUE THREAD

**Issue by michelk** [srid/neuron#520]
Images as symlinks are currently not copied to the output directory. 

```bash
cd zettelkasten
mkdir static
cd static
ln -s ~/Photos/rose.png .
```

Is this intentional? Otherwise I would suggest, that the symlinks are copied dereferenced to the output directory.

**Comment by srid** [comment]
Why should neuron be responsible for dereferencing it? What if the static folder had a symlink to `/`?

**Comment by michelk** [comment]
To be clear, with dereferencing I mean e.g. `rsync -L` where

`-L, --copy-links            transform symlink into referent file/dir` 

> What if the static folder had a symlink to /?

Couldn't we skip the symlink if it points to a directory?

**Comment by srid** [comment]
Yes, if it is a file-target, it should be okay to make neuron copy its contents.

**Comment by srid** [comment]
Closing per https://neuron.zettel.page/next ... to further discuss this idea, [post in Emanote repo](https://github.com/srid/emanote/issues/new/choose).
